multus/multus_test.go

```diff
@@ -48,6 +48,7 @@ type fakePlugin struct {
 	expectedConf    string
 	expectedIfname  string
 	expectedMacAddr string
+	expectedIpAddr  string
```

Comment:
This is golint error in travisCI, so could you please change it to expectedIPAddr?
```
$ golint ./multus/... | grep -v ALL_CAPS
multus/multus_test.go:51:2: struct field expectedIpAddr should be expectedIPAddr
```

Reply:
Thanks, done.